web/client/components/data/featuregrid/filterRenderers/__tests__/DateTimeFilter-test.jsx

```diff
@@ -30,7 +30,7 @@ describe('Test for DateTimeFilter component', () => {
         expect(el).toExist();
     });
     it('render with value', () => {
-        ReactDOM.render(<DateTimeFilter type="date" value={"2017-01-05Z"}/>, document.getElementById("container"));
+        ReactDOM.render(<DateTimeFilter type="date" value="2017-01-05T04:04:04Z"/>, document.getElementById("container"));
```

Comment:
`DateTimeFilter` should be enhanced, or I'm afraid it will not work with dates or times only

Reply:
you mean it should handle partial date or time?

Reply:
btw it is using the utcDateWrapper enhancer